src/lib/pages/backoffice/Document/DocumentDetails/state/actions.js

```diff
@@ -144,3 +146,31 @@ export const requestLoanForPatron = (
     }
   };
 };
+
+export class RequestForPatronMessage extends Component {
+  render() {
+    const { deliveryMethod, loanPid } = this.props;
+    return (
+      <p>
+        <Overridable
+          id="RequestForPatronMessage.DeliveryIcon"
+          deliveryMethod={deliveryMethod}
+        />
+        The loan {loanPid} has been requested.{' '}
+        <Link to={BackOfficeRoutes.loanDetailsFor(loanPid)}>
+          You can now view the loan details.
+        </Link>
+      </p>
+    );
+  }
+}
+
+RequestForPatronMessage.propTypes = {
+  deliveryMethod: PropTypes.string.isRequired,
+  loanPid: PropTypes.string.isRequired,
+};
+
+export default Overridable.component(
```

Comment:
Why not adding a prop, `icon`, instead of `deliveryMethod` and use the `parametrize` overriding? Probably hard to explain here, writing, let's chat tomorrow!